src/components/Board.js

```diff
@@ -16,14 +15,85 @@ class Board extends Component {
     };
   }
 
+  componentDidMount() {
+    axios.get('https://inspiration-board.herokuapp.com/boards/katepond/cards')
+    .then((response) => {
+      this.setState({ cards: response.data });
+    })
+    .catch((error) => {
+      this.setState({
+        error: error.message
+      })
+    });
+  }
+
+  addCard = (card) => {
+    const cards = this.state.cards;
+
+    axios.post('https://inspiration-board.herokuapp.com/boards/katepond/cards', card)
+    .then((response) => {
+    this.componentDidMount();
```

Comment:
This is inefficient to re-fetch all the cards every time you add one to the API.  Instead inspect the `response` object.   It returns the card data for you to use.